unity/CAVE/Assets/Editor/CLI.cs

```diff
@@ -69,156 +65,152 @@ public static void Main()
         }
 
         // Get command line arguments from Python
-        private static string GetXmlPathArg()
+        private static void GetXmlPathArg()
         {
             try
             {
                 string[] args = Environment.GetCommandLineArgs();
                 for (int i = 0; i < args.Length; i++)
                 {
-                    if (args[i] == "--xmlPath") { return args[++i]; }
+                    if (args[i] == "--projectPath") { ProjectPath = args[++i]; }
                 }
-                return null; // No Path given
             }
             catch (Exception e)
             {
                 Debug.Log("Error initializing command line arguments");
                 Debug.LogException(e);
-                return null;
             }
         }
 
-        // Deserialize the xml file into a Story object
-        private static void LoadStory(string xmlPath)
+        // Deserialize the xml file
+        private static void LoadStory()
         {
             try
             {
-                XmlSerializer serializer = new(typeof(Story));
-                using XmlReader reader = XmlReader.Create(xmlPath);
-                XML = (Story)serializer.Deserialize(reader);
+                System.Xml.Serialization.XmlSerializer serializer = new(typeof(Root));
+                using var reader = System.Xml.XmlReader.Create(ProjectPath);
+                RootX = (Root)serializer.Deserialize(reader);
             }
-            catch (FileNotFoundException e)
+            catch (FileNotFoundException)
             {
-                Debug.LogError($"ERROR: File at {xmlPath} not found");
-                Debug.LogException(e);
+                Debug.LogError($"ERROR: File at {ProjectPath} not found");
+                throw;
             }
-            catch (Exception e)
+            catch
             {
-                Debug.LogError($"Error: Deserialization of file at {xmlPath} failed.");
-                Debug.LogException(e);
+                Debug.LogError($"Error: Deserialization of file at {ProjectPath} failed.");
+                throw;
```

Comment:
Likewise, how are exceptions handled by Unity?

Reply:
Unity has it's own sudo console panel where `Debug.Log`, `Debug.Error`, `Debug.Warning`, etc all show up. If it's an actual exception then execution with end and the message is printed as an error